forge.config.js

```diff
@@ -1,85 +1,87 @@
-const packackeJson = require('./package.json')
+const packageJson = require('./package.json')
 
-const { version } = packackeJson;
+const { version } = packageJson;
 const iconDir = path.resolve(__dirname, 'assets', 'icons');
 
-module.exports = {
-  hooks: {
-    generateAssets: require('./tools/generateAssets')
-  },
-  packagerConfig: {
-    name: 'Electron Fiddle',
-    executableName: 'electron-fiddle',
-    asar: true,
-    icon: path.resolve(__dirname, 'assets', 'icons', 'fiddle'),
-    // TODO: FIXME?
-    // ignore: [
-    //   /^\/\.vscode\//,
-    //   /^\/tools\//
-    // ],
-    appBundleId: 'com.electron.fiddle',
-    appCategoryType: 'public.app-category.developer-tools',
-    protocols: [{
-      name: 'Electron Fiddle Launch Protocol',
-      schemes: ['electron-fiddle']
-    }],
-    win32metadata: {
-      CompanyName: 'Electron Community',
-      OriginalFilename: 'Electron Fiddle',
+module.exports = arch => {
+  return {
+    hooks: {
+      generateAssets: require('./tools/generateAssets')
     },
-    osxSign: {
-      identity: 'Developer ID Application: Felix Rieseberg (LT94ZKYDCJ)'
-    }
-  },
-  makers: [
-    {
-      name: '@electron-forge/maker-squirrel',
-      platforms: ['win32'],
-      config: {
-        name: 'electron-fiddle',
-        authors: 'Electron Community',
-        exe: 'electron-fiddle.exe',
-        iconUrl: 'https://raw.githubusercontent.com/electron/fiddle/0119f0ce697f5ff7dec4fe51f17620c78cfd488b/assets/icons/fiddle.ico',
-        loadingGif: './assets/loading.gif',
-        noMsi: true,
-        remoteReleases: '',
-        setupExe: `electron-fiddle-${version}-setup.exe`,
-        setupIcon: path.resolve(iconDir, 'fiddle.ico'),
-        certificateFile: process.env.WINDOWS_CERTIFICATE_FILE,
-        certificatePassword: process.env.WINDOWS_CERTIFICATE_PASSWORD
+    packagerConfig: {
+      name: 'Electron Fiddle',
+      executableName: 'electron-fiddle',
+      asar: true,
+      icon: path.resolve(__dirname, 'assets', 'icons', 'fiddle'),
+      // TODO: FIXME?
+      // ignore: [
+      //   /^\/\.vscode\//,
+      //   /^\/tools\//
+      // ],
+      appBundleId: 'com.electron.fiddle',
+      appCategoryType: 'public.app-category.developer-tools',
+      protocols: [{
+        name: 'Electron Fiddle Launch Protocol',
+        schemes: ['electron-fiddle']
+      }],
+      win32metadata: {
+        CompanyName: 'Electron Community',
+        OriginalFilename: 'Electron Fiddle',
+      },
+      osxSign: {
+        identity: 'Developer ID Application: Felix Rieseberg (LT94ZKYDCJ)'
       }
     },
-    {
-      name: '@electron-forge/maker-zip',
-      platforms: ['darwin']
-    },
-    {
-      name: '@electron-forge/maker-deb',
-      platforms: ['linux'],
-      config: {
-        icon: {
-          scalable: path.resolve(iconDir, 'fiddle.svg')
+    makers: [
+      {
+        name: '@electron-forge/maker-squirrel',
+        platforms: ['win32'],
+        config: {
```

Comment:
It's only this line that has to be a function, not the whole config afaik

Reply:
Aha, I'll fix that.

Reply:
Force-pushed the change: https://github.com/electron/fiddle/commit/0d0a51a900e4f2dafac40e70533e797d20d9d7c3